tournament_results.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+"""
+Get the results a Challonge tournament and use the information to update
+the Smash results spreadsheet.
+
+Pre-Reqs:
```

Comment:
Should we add these to the README as well, or do you think it's getting stuffed?